src/samples/index.md

```diff
@@ -218,6 +218,64 @@ including initializer lists, optional `new` and `const`, redirecting constructor
 `factory` constructors, getters, setters, and much more.
 
 
+## Enums
+
+Enums are a way of enumerating a predefined set of values or instances.
+
+Here is an example of a simple `enum` that defines
+a simple list of predefined planet types:
+
+<?code-excerpt "misc/lib/samples/spacecraft.dart (simple-enum)"?>
+```dart
+enum PlanetType { terrestrial, gas, ice }
+```
+
+Here is an example of an enhanced enum declaration
+that allows you to define a class-like structure
```

Comment:
allows you to define -> defines

I'm not sold on "class-like structure". 
It *is* a class. An enum declaration introduces a class and some instances of that class, and further ensures the class is `const`, sealed and that no-one can create other instances. But it still is a class.)


Maybe:

 Here is an example of an enhanced enum declaration
 of a class describing planets,
 with a defined set of constant instances, 
 namely the planets of our own solar system.

Reply:
I like how this adjustment reads a lot, thanks!